packages/docs/.vitepress/theme/index.ts

```
@@ -3,35 +3,36 @@ import type { App, Component, DefineComponent } from 'vue';
+import SelectDocs                                     from '@/Forms/Select/SelectDocs.vue';
+import TextareaDocs                                   from '@/Forms/Textarea/TextareaDocs.vue';
+import MenuDocs                                       from '@/Navigations/Menu/MenuDocs.vue';
+import NavbarDocs                                     from '@/Navigations/Navbar/NavbarDocs.vue';
+import PaginationDocs                                 from '@/Navigations/Pagination/PaginationDocs.vue';
+import ModalDocs                                      from '@/Overlays/Modal/ModalDocs.vue';
+import PopoverDocs                                    from '@/Overlays/Popover/PopoverDocs.vue';
+import TooltipDocs                                    from '@/Overlays/Tooltip/TooltipDocs.vue';
+import Playground                                     from '@/Playground/Playground.vue';
+import CreateFlyonUIVueAppDocs                        from '@/QuickStart/CreateFlyonUIVueApp/CreateFlyonUIVueAppDocs.vue';
+import TableDocs                                      from '@/Tables/Table/TableDocs.vue';
+import { FoKeyboard, FoSelectThemeController, vMask } from 'flyonui-vue';
```

ESLint

[error] Expected "flyonui-vue" (external) to come before "@/Tables/Table/TableDocs.vue" (internal).

(perfectionist/sort-imports)